LoopCaregiverKit/Sources/LoopCaregiverKitUI/LatestGlucoseRectangularView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LatestGlucoseRectangularView.swift
+//  LoopCaregiverWidgetExtension
+//
+//  Created by Auggie Fisher on 1/24/24.
+//
+
+import HealthKit
+import LoopCaregiverKit
+import LoopKit
+import SwiftUI
+
+public struct LatestGlucoseRectangularView: View {
+
+    
+    public let viewModel: WidgetViewModel
+    @Environment(\.colorScheme) var colorScheme
+    
+    public init(viewModel: WidgetViewModel) {
+        self.viewModel = viewModel
+    }
+    
+    public var body: some View {
+        
+        HStack (spacing: 10) {
+            VStack {
+                //BG number
+                Text(viewModel.currentGlucoseNumberText)
+                    .foregroundStyle(egvColor)
+                    .strikethrough(viewModel.isGlucoseStale)
+                    .font(.system(size: 65))
+            }
+            VStack (spacing: 0) {
+                HStack {
```

Comment:
No need to change but just a tip - if you find you are commenting in swift UI to explain what the views are - I find it nice to just make vars that create the view. For example: bgView, trendArrowView, bgDeltaView, minutesSinceUpdateView.

That var is self-documenting. Then your view body becomes really compact. If you take a look at the OverrideView, you can see an example.

Reply:
I love that! Thanks for this tip!